LykkeApi2/Controllers/OperationsController.cs

```diff
@@ -10,39 +14,88 @@ namespace LykkeApi2.Controllers
     [Route("api/operations")]
     public class OperationsController : Controller
     {
+        private readonly IOperationsAPI _operationsApi;
+        private readonly IRequestContext _requestContext;
+
+        public OperationsController(IOperationsAPI operationsApi, IRequestContext requestContext)
+        {
+            _operationsApi = operationsApi;
+            _requestContext = requestContext;
+        }
+
+        /// <summary>
+        /// Get operation by id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
         [HttpGet]
         [Route("{id}")]
         [ApiExplorerSettings(GroupName = "Operations")]
-        public IActionResult Get(Guid? id)
+        public async Task<IActionResult> Get(Guid? id)
```

Comment:
Maybe just Guid would be better instead of Guid?
As far as I see you are checking if the parameter has value or not so what's the point of allowing nulls here?

Reply:
fixed